packages/patternfly-3/patternfly-react/src/components/Overlay/__mocks__/mockOverlayManager.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/no-find-dom-node */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { Button } from '../../Button';
+import { Popover } from '../../Popover';
+import { Overlay } from '../index';
+
+export class MockOverlayManager extends React.Component {
+  constructor() {
+    super();
+    this.buttonRef = React.createRef();
+    this.state = { showOverlay: false };
+  }
+  toggleOverlay = () => {
+    this.setState({ showOverlay: !this.state.showOverlay });
+  };
+  render() {
+    const { placement } = this.props;
+    const { showOverlay } = this.state;
+
+    return (
+      <div>
+        <Button bsStyle="primary" bsSize="large" onClick={this.toggleOverlay} ref={this.buttonRef}>
+          Click Here
+        </Button>
+        <Overlay
+          target={() => ReactDOM.findDOMNode(this.buttonRef.current)}
```

Comment:
Is it possible to get the DOM node already from `this.buttonRef.current` instead of using `ReactDOM.findDOMNode` ?

Reply:
I don't think so, see the official doc from react-bootstrap: https://react-bootstrap.github.io/components/overlays/#custom-overlays-overlay

Reply:
See: https://reactjs.org/docs/refs-and-the-dom.html#exposing-dom-refs-to-parent-components

Reply:
In the link you sent they seem to refer to child nodes,
and they also say about `findDOMNode` that: ` your last option is to use findDOMNode(), but it is discouraged and deprecated in StrictMode.`

Reply:
I'm confused, isn't it the current case? Do I have another access to the actual dom-node?

Reply:
just changed the overlay target into: `target={this.buttonRef.current}`,
I checked, it works :)

Reply:
Awesome :fireworks: 
I'm not sure why the react-bootstrap example is so outdated, was pretty sure there is a reason for it.